.github/workflows/conda-build.yml

```diff
@@ -15,46 +15,42 @@ jobs:
         include:
           - CONFIG: linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10
             SHORT_CONFIG: linux_64_c_compiler_version10cuda_c_h409a79459c
-            UPLOAD_PACKAGES: False
+            UPLOAD_PACKAGES: True
             DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.2
             os: ubuntu
           - CONFIG: linux_64_c_compiler_version12cuda_compiler_version12.0cxx_compiler_version12
             SHORT_CONFIG: linux_64_c_compiler_version12cuda_c_hae9b567e31
-            UPLOAD_PACKAGES: False
+            UPLOAD_PACKAGES: True
             DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
             os: ubuntu
           - CONFIG: linux_64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12
             SHORT_CONFIG: linux_64_c_compiler_version12cuda_c_hb9e6816263
-            UPLOAD_PACKAGES: False
+            UPLOAD_PACKAGES: True
             DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
             os: ubuntu
-          - CONFIG: linux_aarch64_c_compiler_version10cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10
-            SHORT_CONFIG: linux_aarch64_c_compiler_version10c_hb0ea797ce5
-            UPLOAD_PACKAGES: False
+          - CONFIG: linux_aarch64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10
+            SHORT_CONFIG: linux_aarch64_c_compiler_version10c_h6023db96ca
+            UPLOAD_PACKAGES: True
             DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.2
             os: ubuntu
-          - CONFIG: linux_aarch64_c_compiler_version12cuda_compilerNonecuda_compiler_versionNonecxx_compiler_version12
-            SHORT_CONFIG: linux_aarch64_c_compiler_version12c_h07c7302bdb
-            UPLOAD_PACKAGES: False
+          - CONFIG: linux_aarch64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12
+            SHORT_CONFIG: linux_aarch64_c_compiler_version12c_h21a2ab6f58
+            UPLOAD_PACKAGES: True
             DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
             os: ubuntu
           - CONFIG: win_64_cuda_compiler_versionNone
             SHORT_CONFIG: win_64_cuda_compiler_versionNone
-            UPLOAD_PACKAGES: False
+            UPLOAD_PACKAGES: True
             os: windows
           - CONFIG: osx_64_
             SHORT_CONFIG: osx_64_
-            UPLOAD_PACKAGES: False
+            UPLOAD_PACKAGES: True
             os: macos
           - CONFIG: osx_arm64_
             SHORT_CONFIG: osx_arm64_
-            UPLOAD_PACKAGES: False
+            UPLOAD_PACKAGES: True
             os: macos
     steps:
-    - name: Delete huge unnecessary tools folder
-      run: rm -rf /opt/hostedtoolcache
-      if: matrix.os == 'ubuntu'
-
```

Comment:
Let's readd this. @hcho3 added it since we are running into space limitations on VM